Assets/Scripts/PizzaGame/Ingredient.cs

```diff
@@ -11,6 +13,11 @@ public class Ingredient : MonoBehaviour
         public IngredientType Type => type;
 
         public bool IsInPlace { get; set; }
+        private AudioSource audioSource;
+
+        private void Start() {
+            audioSource = GetComponent<AudioSource>();
+        }
```

Comment:
Da sich unsere Kamera quasi nicht bewegt, könntest Du glaube ich theoretisch die Sounds einfach alle vom SoundEffectManager abspielen lassen, weil die Unterschiede in der Position nicht wirklich hörbar sind 😅 Aber Du kannst es auch so lassen, ist ja letztendlich egal.
Wenn Du die AudioSource auf den Ingredients lässt, kannst Du der Klasse noch n `[RequireComponent(AudioSource)]`-Attribut geben. Dann kann die AudioSource nicht mehr gelöscht werden, solange ein Ingredient-Script auf nem GameObject ist und `GetComponent<>()` returnt nie null

Reply:
Wir könnten den Köchen noch die Soundeffekte geben xD aber das wäre zu viel des Guten

Reply:
Aber hast recht, ich dachte, man hört kleine Unterschiede.